spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/repository/query/DatastorePageable.java

```diff
@@ -22,33 +22,44 @@
 import org.springframework.data.domain.Pageable;
 
 /**
+ * A pageable implementation for Cloud Datastore that uses the cursor for efficient reads.
+ *
  * @author Dmitry Solomakha
+ * @author Chengyuan Zhao
  */
 public class DatastorePageable extends PageRequest {
-	private final Cursor cursor;
+	private final String urlSafeCursor;
```

Comment:
Maybe add `getUrlSafeCursor` method?

Reply:
They can retrieve the urlSafeString from the Cursor actually. In fact we don't have the string aspect of it visible at all to the outside. I prefer to keep it only in terms of Cursor externally.

Reply:
Actually let me add that getter. Perhaps a public getter will cause the serializer to pick it up. Stay tuned

Reply:
fixed :)